### ISSUE TITLE
test: test_message failed but is marked as success

### ISSUE THREAD

**Issue by alfredh** [baresip/baresip#2993]
git HEAD:

```
[ RUN      ] test_message (rx thread)
selftest: ASSERT_EQ: /home/alfredh/git/baresip/test/message.c:127: send_resp_handler_488(): expected=488(0x1e8), actual=403(0x193)

TEST_ERR: /home/alfredh/git/baresip/test/message.c:242: (Invalid argument [22])
selftest: ASSERT_EQ: /home/alfredh/git/baresip/test/message.c:127: send_resp_handler_488(): expected=488(0x1e8), actual=403(0x193)

TEST_ERR: /home/alfredh/git/baresip/test/message.c:242: (Invalid argument [22])
[       OK ]
```

**Comment by maximilianfridrich** [comment]
Fixed in #2996.

**Comment by maximilianfridrich** [comment]
Oops, didn't see your PR @alfredh. I closed mine.
